modules/ROOT/pages/getting-started.adoc

```diff
@@ -2,16 +2,16 @@
 
 == Introduction
 
-=== Update Streams
+=== Streams
 
-There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well.
+There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well and provide feedback.
 
-Each stream has a canonical URL representing its current state in JSON format.  For example, the URL for the stable stream is: https://builds.coreos.fedoraproject.org/streams/stable.json
+Each stream has a canonical URL representing its current state in JSON format, known as "stream metadata".  For example, the stream metadata URL for `stable` is: https://builds.coreos.fedoraproject.org/streams/stable.json.
```

Comment:
Does the added final period here render as part of the URL? (At least [Refined GitHub](https://github.com/sindresorhus/refined-github/) gets thrown off by it here.)

Reply:
Fixed.